common/views/components/IIIFPresentationPreview/IIIFPresentationPreview.js

```diff
@@ -0,0 +1,387 @@
+// @flow
+import { type IIIFManifest } from '@weco/common/model/iiif';
+import { getCanvases, getManifestViewType } from '@weco/common/utils/iiif';
+import NextLink from 'next/link';
+import styled from 'styled-components';
+import { iiifImageTemplate } from '@weco/common/utils/convert-image-uri';
+import { grid } from '@weco/common/utils/classnames';
+import { type Node, useEffect, useState, useContext, useRef } from 'react';
+import { trackEvent } from '@weco/common/utils/ga';
+import ManifestContext from '@weco/common/views/components/ManifestContext/ManifestContext';
+import Button from '@weco/common/views/components/Buttons/Button/Button';
+import BetaMessage from '@weco/common/views/components/BetaMessage/BetaMessage';
+import IIIFResponsiveImage from '@weco/common/views/components/IIIFResponsiveImage/IIIFResponsiveImage';
+import WobblyRow from '@weco/common/views/components/WobblyRow/WobblyRow';
+import Space from '../styled/Space';
+import useOnScreen from '@weco/common/hooks/useOnScreen';
+
+const MultiVolumeContainer = styled.div`
+  box-shadow: ${props =>
+    props.isOnScreen
+      ? `12px 12px 0px 0px ${props.theme.colors.yellow}`
```

Comment:
Should we be theming box-shadows to avoid design creep?

Reply:
Maybe, I wouldn't bother reviewing the IIIFPresentationPreview or IIIFImage Preview though,  we're going to get rid of them ASAP.

Reply:
We've got a [.shadow](https://github.com/wellcomecollection/wellcomecollection.org/blob/master/common/styles/utilities/_root-scope-classes.scss#L529) class, for the record.